### PR TITLE
add phantom setting. [To fix no proxy setting if html is url]

### PR DESCRIPTION
**Pull request by bcguan2008**
When i used uncss to process a url. but no setting about proxy. 

phridge had provided the options. we can only add this very easy.

### CODE REVIEW

File: src/phantom.js [hunk 1]
```diff
@@ -18,18 +18,15 @@ var phantom;
...
         phantom = instance;
         return null;
     }
 
     // Convert to bluebird promise
     return new promise(function (resolve) {
-        resolve(phridge.spawn({
-            ignoreSslErrors: 'yes',
-            sslProtocol: 'any'
-        }));
```
Comment (pconerly) [inline src/phantom.js]: I'm not a maintainer, but you should keep these settings as defaults. You can do this with:

```
options = _.assign({
            ignoreSslErrors: 'yes',		
            sslProtocol: 'any'
}, options);
```

File: src/uncss.js [hunk 2]
```diff
@@ -213,11 +213,12 @@ function init(files, options, callback) {
         media: [],
         timeout: 0,
         report: false,
-        ignoreSheets: []
+        ignoreSheets: [],
+        httpSetting: {},
```
Comment (pconerly) [inline src/uncss.js]: It might be more accurate to call this `phantomSettings` or `phridgeSettings`

Comment (pconerly) [inline src/uncss.js]: Also, FYI, the project's linting rules failed the CI tests because there's a trailing comma on this line

**Comment by XhmikosR** [comment]
@mikelambert: shouldn't this be closed now with jsdom?

**Pull request closed without merging**